interchain/framework-clone-testing/src/common.rs

```diff
@@ -0,0 +1,63 @@
+use abstract_client::{AbstractClient, Environment};
+use abstract_interface::{Abstract, VCExecFns};
+use cosmwasm_std::{coins, Addr};
+use cw_orch::prelude::*;
+use cw_orch::{contract::Deploy, daemon::ChainInfo};
+use cw_orch_clone_testing::CloneTesting;
+
+/// Returns a shared tokio runtime for all tests
+pub fn rt() -> &'static tokio::runtime::Runtime {
+    lazy_static::lazy_static! {
+        static ref RT: tokio::runtime::Runtime = tokio::runtime::Builder::new_multi_thread()
+        .enable_all()
+        .build()
+        .expect("Should create a tokio runtime");
+    }
+    &RT
+}
+
+/// Sets up the CloneTesting for chain.
+/// Returns the abstract deployment and sender (=mainnet admin)
+pub fn setup(
+    chain: ChainInfo,
+    sender: &str,
+) -> anyhow::Result<(Abstract<CloneTesting>, CloneTesting)> {
+    let _ = env_logger::builder().is_test(true).try_init();
+    let sender = Addr::unchecked(sender);
+    // Run migration tests against Juno mainnet
+    // We set the state file to be able to clone test
+    std::env::set_var("STATE_FILE", "../scripts/state.json");
+    let mut app = CloneTesting::new(rt(), chain)?;
+    app.set_sender(sender.clone());
+
+    let abstr_deployment = Abstract::load_from(app.clone())?;
+    Ok((abstr_deployment, app))
+}
+
+/// Sets up the CloneTesting for chain.
+/// Returns the abstract client
+pub fn load_abstr(chain: ChainInfo, sender: Addr) -> anyhow::Result<AbstractClient<CloneTesting>> {
+    let _ = env_logger::builder().is_test(true).try_init();
+    std::env::set_var("STATE_FILE", "../scripts/state.json");
```

Comment:
Is this still required, even with the new crate we use to set the path of the state file?

Reply:
Shouldn't be, let's see if CI fails

Reply:
@CyberHoward this fix is not present in the cw-orch-interchain tag version: https://github.com/AbstractSDK/cw-orchestrator/pull/333 so daemonstate doesn't creates empty state and clone test will try to read daemonstate file regardless

Reply:
Nvm we need this line because clone-testing requires existing `state.json` file, since it opens `DaemonState` in read-only mode and fails at first read